dotnet/src/Planners/Planners.Core/Extensions/PromptTemplateConfigExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the namespace of IKernel
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.SemanticFunctions;
+#pragma warning restore IDE0130
```

Comment:
nit: Warning disable can be located right before namespace declaration. Also, I'm wondering, should it be in `Microsoft.SemanticKernel.SemanticFunctions` namespace or `Microsoft.SemanticKernel.Planners` as other extension classes in `Planners.Core` project?

```suggestion
using Microsoft.SemanticKernel.AI;

#pragma warning disable IDE0130
// ReSharper disable once CheckNamespace - Using the namespace of IKernel
namespace Microsoft.SemanticKernel.SemanticFunctions;
#pragma warning restore IDE0130
```

Reply:
I'd expect extensions for `PromptTemplateConfig` to be in it's namespace